src/main/java/com/maddyhome/idea/vim/newapi/IjVimInjector.kt

```diff
@@ -212,3 +217,19 @@ internal class IjVimInjector : VimInjectorBase() {
   override val editorGroup: VimEditorGroup
     get() = service<EditorGroup>()
 }
+
+// While these functions don't use VimInjector, they mirror VimInjector.globalOptions() so it's good for discovery
+// They are specific to the IntelliJ implementation of VimOptionGroup. We can't rely on upcasting injector.optionGroup
+// because we override it in tests to track what options are used.
+
+/**
+ * Convenience function to get the IntelliJ implementation specific global option accessor
+ */
+@Suppress("UnusedReceiverParameter")
+public fun VimInjector.globalIjOptions(): GlobalIjOptions = getGlobalIjOptions()
+
+/**
+ * Convenience function to get the IntelliJ implementation specific option accessor for the given editor's scope
+ */
+@Suppress("UnusedReceiverParameter")
+public fun VimInjector.ijOptions(editor: VimEditor): EffectiveIjOptions = getEffectiveIjOptions(editor)
```

Comment:
I'm not too keen on this, but we can't upcast `VimOptionGroup` to IntelliJ's `OptionGroup` because we re-implement the `VimOptionGroup` in tests. An alternative would be to create an `IjVimOptionGroup` interface to access IntellIJ specific options, and have the test class also implement this interface. What do you think?